CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ### Fixed
 - Technical - Fix test expect files extensions.
 - Command Generate - Do not declare the "id" field on tables having a `BIGINT` "id" column as primary key. 🛡
+- Command Generate - Fix issue where a conflict between regular fields and references alias could happen, and prevent the server from starting. 
```

Comment:
[Suggestion] 
```suggestion
- Command Generate - Fix issue where a conflict between regular fields and references alias could prevent the server from starting. 
```

Not sure though. Maybe yours is better

Reply:
👍